src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowsServices/Properties/Resources.resx

```diff
@@ -0,0 +1,210 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
👀 we actually didn't really have any localized components before this. Glad that it just worked?

Reply:
#124 -- I'm not sure if it is actually localized or not / how that pipeline works. I think it is?